addons/rating/models/rating_mixin.py

```diff
@@ -61,7 +61,7 @@ def _compute_rating_stats(self):
 
     def _search_rating_avg(self, operator, value):
         if operator not in rating_data.OPERATOR_MAPPING:
-            raise NotImplementedError('This operator %s is not supported in this search method.' % operator)
+            raise NotImplementedError(_('This operator %s is not supported in this search method.'), operator)
```

Comment:
I had confusion about adding a variable to _(), but then I saw some other changes like [this](https://github.com/odoo/odoo/blob/16.0/addons/account/models/account_bank_statement_line.py#L305C1-L306C67).
so because of that, I did it like this.
so what should I do now? :thinking: 
should I change it as you said or not? :confused:

Reply:
Hey, thanks for the quick reply :grin: 
What I suggested is actually what you are showing me in that file :innocent: the variable operator is inside the _() while in your change it's not.

Reply:
oh, Sorry I misunderstood it. :sweat: 
I really need an eye checkup now :laughing: